tests/framework/Optimizers/Stochastic/stoch_parabola.py

```diff
@@ -17,7 +17,8 @@ def random(scale=0.5,loc=-1.0):
   return scale*(2.*np.random.rand()+loc)
 
 def evaluate(values):
-  return base(values) + random()
+  ran = random()/10.0
```

Comment:
Because in this test the noise introduced by the random noise was comparable with the objective function evaluations. This test was not converging but just terminated when the max number of iterations were reached. Now it converges since I reduced the noise to 10% of the previous one